x-pack/test/functional/apps/security/management.js

```diff
@@ -148,7 +156,9 @@ export default function ({ getService, getPageObjects }) {
           await PageObjects.security.clickSaveEditUser();
 
           await PageObjects.settings.navigateTo();
-          await testSubjects.click('users');
+          await testSubjects.click('roles');
```

Comment:
Surprised this test passes. 

Why did you change 'users' to 'roles'? Shouldn't we check that the newly created user appears on the user listing page?